chainstate/test-suite/src/tests/chainstate_accounting_storage_tests.rs

```diff
@@ -154,9 +177,17 @@ fn store_pool_data_and_balance(#[case] seed: Seed) {
             1
         );
 
+        let (genesis_pool_id, genesis_pool_data) = get_genesis_pool_data();
+
         let expected_tip_storage_data = pos_accounting::PoSAccountingData {
```

Comment:
I'd create the same data for sealed storage to check that only genesis info is there. Right now the check is weak. In other tests as well